pyproject.toml

```diff
@@ -15,7 +15,6 @@ dependencies = [
 
     # Tokenization
     "blobfile",
-    "sentencepiece",
```

Comment:
Let's remove the following as well
https://github.com/pytorch/torchtitan/blob/main/.ci/docker/requirements.txt#L5